tools/cldr-apps/src/main/java/org/unicode/cldr/web/STFactory.java

```diff
@@ -1754,6 +1754,10 @@ protected CLDRFile handleMake(String localeID, boolean resolved, DraftStatus mad
         return get(localeID).getFile(resolved);
     }
 
+    public CLDRFile make(String loc) {
```

Comment:
This new single-parameter make() is for both DataSection and Dashboard. It's equivalent to the current code, but better since we want DataSection and Dashboard to be consistent, and the current code arbitrarily uses equivalent 2-parameter and 3-parameter versions of make().